src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -751,6 +751,13 @@ public function deprecateParameter(string $name, string $package, string $versio
      */
     public function compile(bool $resolveEnvPlaceholders = false)
     {
+        foreach ($this->extensions as $extension) {
+            // @deprecated remove if statement in Symfony 7
+            if (method_exists($extension, 'build')) {
+                $extension->build($this);
+            }
```

Comment:
… else trigger a deprecation? The method will be enforced by `ExtensionInterface` in v7, right?

Reply:
The DebugClassLoader automatically triggers a deprecation when implementing `ExtensionInterface` without this method (thanks to the `@method build` annotation).